src/core/cdr/src/dds_cdrstream.c

```diff
@@ -4439,6 +4865,7 @@ static void dds_stream_read_key_impl (dds_istream_t * __restrict is, char * __re
     case DDS_OP_VAL_2BY: *((uint16_t *) dst) = dds_is_get2 (is); break;
```

Comment:
[lines 4723 and 4805] `WCHAR`, `WSTR` and `BWSTR` cases missing (cast `subtype` and `type` to enum so that the compiler shows a warning?)